test/CppTest/CppTest.vcxproj

```diff
@@ -332,17 +332,17 @@
   <ImportGroup Label="ExtensionTargets">
     <Import Project="..\..\packages\Microsoft.Windows.ImplementationLibrary.1.0.200519.2\build\native\Microsoft.Windows.ImplementationLibrary.targets" Condition="Exists('..\..\packages\Microsoft.Windows.ImplementationLibrary.1.0.200519.2\build\native\Microsoft.Windows.ImplementationLibrary.targets')" />
     <Import Project="..\..\packages\Microsoft.Windows.CppWinRT.2.0.200703.9\build\native\Microsoft.Windows.CppWinRT.targets" Condition="Exists('..\..\packages\Microsoft.Windows.CppWinRT.2.0.200703.9\build\native\Microsoft.Windows.CppWinRT.targets')" />
-    <Import Project="..\..\packages\Taef.Redist.Wlk.10.52.200221001\build\Taef.Redist.Wlk.targets" Condition="Exists('..\..\packages\Taef.Redist.Wlk.10.52.200221001\build\Taef.Redist.Wlk.targets')" />
-    <Import Project="..\..\packages\Taef.TestAdapter.10.57.201103002-develop\build\Taef.TestAdapter.targets" Condition="Exists('..\..\packages\Taef.TestAdapter.10.57.201103002-develop\build\Taef.TestAdapter.targets')" />
+    <Import Project="..\..\packages\Microsoft.Taef.Redist.Wlk.10.58.210126003-develop\build\Taef.Redist.Wlk.targets" Condition="Exists('..\..\packages\Microsoft.Taef.Redist.Wlk.10.58.210126003-develop\build\Taef.Redist.Wlk.targets')" />
```

Comment:
>Redist.Wlk [](start = 51, length = 10)

i think this contains testadapter, doesn't it?

Reply:
i think Taef.Redit.Wlk is the only package TAEF team agreed to distribute publicly. MRT unit test (BaseUnitTests) doesn't need the extra Taef.TestAdapter package, and it runs inside VS.